engine/consensus/sealing/engine.go

```diff
@@ -43,22 +43,21 @@ type (
 // Purpose of this struct is to provide an efficient way how to consume messages from network layer and pass
 // them to `Core`. Engine runs 2 separate gorourtines that perform pre-processing and consuming messages by Core.
 type Engine struct {
-	unit                                 *engine.Unit
-	core                                 sealing.SealingCore
-	log                                  zerolog.Logger
-	me                                   module.Local
-	headers                              storage.Headers
-	payloads                             storage.Payloads
-	cacheMetrics                         module.MempoolMetrics
-	engineMetrics                        module.EngineMetrics
-	pendingApprovals                     engine.MessageStore
-	pendingRequestedApprovals            engine.MessageStore
-	pendingFinalizationEvents            *fifoqueue.FifoQueue
-	pendingIncorporatedResults           *fifoqueue.FifoQueue
-	notifier                             engine.Notifier
-	messageHandler                       *engine.MessageHandler
-	requiredApprovalsForSealConstruction uint
```

Comment:
removed struct field `requiredApprovalsForSealConstruction` as the value is only needed at construction time